### PR TITLE
AccordionLabel does not match the theme key. 

### LINKED ISSUE #2217: `AccordionLabel` does not match the theme key.
### Description

Currently, the key of the theme is `button`. Since it is also a `button` in role, rename `AccordionLabel` to `AccordionButton`.

### Problem Statement/Justification

It is wrong if the names do not match or are not appropriate for the role.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

If you rename it to `AccordionButton`, you will need to change the name of the `props`, the description of the `JSDoc`, and everywhere else.

### PR DESCRIPTION
**Pull request by kaitobq**
Closes #2217  

## Description

Renamed AccordionLabel to AccordionButton and modified props and docs accordingly.

### CODE REVIEW

**Comment by kaitobq** [comment]
I have run pnpm storybook, pnpm build, pnpm test and verified there are no errors.
Please let me know if there is anything else I need to do.
Please let me know if what I am committing to is different from what I am doing.

**Review by 108yen — CHANGES_REQUESTED** [review]
1. Branch is incorrect, please correct.
2. Please provide a description of the PR changes.

**Comment by kaitobq** [comment]
@108yen 
Sorry I did not read Contributing.md thoroughly.
Can I close this pull request and create the pull request again?

**Comment by 108yen** [comment]
@kaitobq 
That is not necessary.
You can edit the PR description as you go along.

**Comment by kaitobq** [comment]
@108yen 
Noted. I thought the branch name was the problem.
I have added the Description.

File: docs/contents/components/disclosure/accordion/index.mdx
Comment (108yen) [inline docs/contents/components/disclosure/accordion/index.mdx:1]: The `label` remains.

File: docs/contents/components/disclosure/accordion/props.mdx
Comment (108yen) [inline docs/contents/components/disclosure/accordion/props.mdx:1]: The `label` remains.

**Review by 108yen — CHANGES_REQUESTED** [review]
The branch has been changed and is fine.
Run `changeset` and select `major` to record the changes.

File: packages/components/accordion/src/accordion-item.tsx
Comment (108yen) [inline packages/components/accordion/src/accordion-item.tsx:1]: Please change classname.
```diff tsx
-className={cx("ui-accordion__item", className)}
+className={cx("ui-accordion__button", className)}
```

**Review by 108yen — CHANGES_REQUESTED** [review]

File: packages/components/accordion/src/accordion-button.tsx [hunk 1]
```diff
@@ -61,7 +62,7 @@ export const AccordionLabel = forwardRef<AccordionLabelProps, "button">(
     return (
       <ui.div role="heading" aria-level="3" {...containerProps}>
         <ui.button
-          {...getLabelProps(rest, ref)}
+          {...getButtonProps(rest, ref)}
           className={cx("ui-accordion__label", className)}
```
Comment (108yen) [inline packages/components/accordion/src/accordion-button.tsx:66]: Please change classname.
```diff tsx
-className={cx("ui-accordion__label", className)}
+className={cx("ui-accordion__button", className)}
```

**Comment by kaitobq** [comment]
@108yen 
I replaced label with button after running changeset, do I need to run changeset again?

**Review by 108yen — APPROVED** [review]
No need.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**